api/core/app/apps/base_app_generate_response_converter.py

```diff
@@ -26,7 +26,10 @@ def convert(cls, response: Union[
             else:
                 def _generate():
                     for chunk in cls.convert_stream_full_response(response):
-                        yield f'data: {chunk}\n\n'
+                        if chunk.startswith('event:'):
```

Comment:
Currently, only "ping" is a special type. Here we can directly change it to a check for whether it is a "ping".